### PR TITLE
Fix inconsistent UX of delete operation

### LINKED ISSUE #602: Inconsistent UX of Delete operation
We would need to use the same confirmation dialods for both delete buttons. Currently the one under **Actions**

delete

uses

confirm

while I assume that the proper one would be

proper

### PR DESCRIPTION
**Pull request by reggie7**
Fix for #602

### CODE REVIEW

File: admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/explorer/template.vue [hunk 1]
```diff
@@ -653,27 +648,34 @@ export default {
...
+                            resolve()
                         }
-                    }
+                    })
                 })
             },
 
+            deletePage: function(me, target) {
+                const { resourceType, path } = target
+                let action = 'folder'
```
Comment (flx-sta) [inline admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/explorer/template.vue]: I thinks it's a bit confusing here that `action` is the name of the variable when it is passed as `type` to `handleDelete()`

**Review by flx-sta — CHANGES_REQUESTED** [review]
Only the small `action -> Type` change please

File: admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/explorer/template.vue (hunk 1 above)
Comment (reggie7) [reply admin-base/ui.apps/src/main/content/jcr_root/apps/admin/components/explorer/template.vue]: Thanks, good catch!

**Review by flx-sta — APPROVED** [review]

**Comment by flx-sta** [comment]
I've wrote an end-to-end test for this since it's an interessting scenario for the page-object pattern.
Because `PagesPage`, `ObjectsPage`, `AssetsPage` and `TemplatesPage` would be very similar and to prevent this class extension is used

https://github.com/Felix-Puetz/pcms-testing/pull/2

**Review by reusr1 — APPROVED** [review]

**Pull request merged**